packages/junit/junit.2.2.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Louis Roché <louis@louisroche.net>"
+authors: "Louis Roché <louis@louisroche.net>"
+homepage: "https://github.com/Khady/ocaml-junit"
+bug-reports: "https://github.com/Khady/ocaml-junit/issues"
+license: "LGPL-3.0-or-later WITH OCaml-LGPL-linking-exception"
+dev-repo: "git+https://github.com/Khady/ocaml-junit.git"
+doc: "https://khady.github.io/ocaml-junit/"
+tags: ["junit" "jenkins"]
+depends: [
+  "dune" {>= "3.0"}
+  "ptime"
+  "tyxml" {>= "4.0.0"}
+  "odoc" {with-doc & >= "1.1.1"}
+  "ocamlformat" {= "0.27.0" & with-dev-setup}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "-j" jobs] {with-doc}
```

Comment:
This looks duplicated, missing an alias?

Reply:
good catch, thanks